src/EditorFeatures/Test2/Rename/RenameTestHelpers.vb

```diff
@@ -97,6 +113,16 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
             Next
         End Function
 
+        Public Sub VerifyFileRename(workspace As TestWorkspace, newIdentifierName As String)
+            Dim renamedDocuments = workspace.Documents.Select(Function(d) d.Id)
+
+            For Each documentId In renamedDocuments
+                Dim document = workspace.CurrentSolution.GetDocument(documentId)
+                Dim expectedName = Path.ChangeExtension(newIdentifierName, Path.GetExtension(document.Name))
```

Comment:
No, [Path.GetExtensions](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.getextension?redirectedfrom=MSDN&view=netframework-4.8#System_IO_Path_GetExtension_System_String_) does last `.` only. That means the option will be disabled because the type doesn't match the file